app/components/viral/card/header_component.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Viral
+  module Card
+    # Header component for the card
+    class HeaderComponent < Component
+      attr_reader :title
+
+      renders_many :actions
+
+      def initialize(title: '', **system_arguments)
+        @title = title
+        @system_arguments = system_arguments
+        @system_arguments[:classes] = class_names(
+          @system_arguments[:classes],
+          'p-4'
+        )
+      end
+
+      def simple?
```

Comment:
Let's rename this to something a bit more clear of what it's for

Reply:
Updated in 4eeca1a97e13f85aa9e1ce703fbb47dd4a348197

Reply:
Thanks!